app/src/index.js

```diff
@@ -40,6 +41,7 @@ import thunk from 'redux-thunk';
 
 const globalExceptionHandler = (error, isFatal) => {
   if (error && NativeModules.Firebase) {
+    console.log(error);
```

Comment:
Didn't prettier/linter get added?

Reply:
It did. But it looks like the precommit script doesn't fire up if I run the commit from the top-level folder. I'll look into fixing this.